manifests/pipecd/templates/deployment.yaml

```diff
@@ -90,7 +90,7 @@ spec:
           imagePullPolicy: IfNotPresent
           args:
           - server
-{{- if or .Values.quickstart .Values.api.args.insecureCookie }}
+{{- if or .Values.quickstart .Values.api.args.insecureCookie (eq (int .Values.service.port) 80 ) }}
```

Comment:
```
{{- if not .Values.api.args.secureCookie }}
- --insecure-cookie=true
{{- end }}
```

Reply:
And please update the doc at 
https://github.com/pipe-cd/pipe/blob/master/docs/content/en/docs/operator-manual/control-plane/installation.md

Reply:
I addressed it. PTAL :pray: